getting_started.md

```diff
@@ -18,7 +18,7 @@ sudo apt-get install android-tools-adb android-tools-fastboot autoconf \
     libftdi-dev libglib2.0-dev libhidapi-dev libncurses5-dev \
     libpixman-1-dev libssl-dev libstdc++6:i386 libtool libz1:i386 make \
     mtools netcat python-crypto python-serial python-wand unzip uuid-dev \
-    xdg-utils xterm xz-utils zlib1g-dev ccache
+    xdg-utils xterm xz-utils zlib1g-dev ccache repo gcc-aarch64-linux-gnu
```

Comment:
You shouldn't need to install gcc-aarch64-linux-gnu. This should be created with `make toolchains` and then adding "<tc_folder>/toolchains/aarch64/bin" and "<tc_folder>/toolchains/aarch32/bin" to your path.